db/schema.rb

```diff
@@ -0,0 +1,61 @@
+# This file is auto-generated from the current state of the database. Instead
+# of editing this file, please use the migrations feature of Active Record to
+# incrementally modify your database, and then regenerate this schema definition.
+#
+# This file is the source Rails uses to define your schema when running `bin/rails
+# db:schema:load`. When creating a new database, `bin/rails db:schema:load` tends to
+# be faster and is potentially less error prone than running all of your
+# migrations from scratch. Old migrations may fail to apply correctly if those
+# migrations use external dependencies or application code.
+#
+# It's strongly recommended that you check this file into your version control system.
+
+ActiveRecord::Schema[7.0].define(version: 2022_12_09_133538) do
+  # These are extensions that must be enabled in order to support this database
+  enable_extension "plpgsql"
+
+  create_table "comments", force: :cascade do |t|
+    t.string "text"
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+    t.bigint "author_id"
+    t.bigint "post_id"
+    t.index ["author_id"], name: "index_comments_on_author_id"
+    t.index ["post_id"], name: "index_comments_on_post_id"
+  end
+
+  create_table "likes", force: :cascade do |t|
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+    t.bigint "author_id"
+    t.bigint "post_id"
+    t.index ["author_id"], name: "index_likes_on_author_id"
+    t.index ["post_id"], name: "index_likes_on_post_id"
+  end
+
+  create_table "posts", force: :cascade do |t|
```

Comment:
- [ ] Good job so far I'm able to create users and add some comments via `rails c` but I'm getting errors when I tried to create posts written by one of the users I create. Please fix it in order to make your app work as expected :+1: 
![Screenshot from 2022-12-09 23-42-58](https://user-images.githubusercontent.com/6615394/206812789-1562a15f-798f-4f14-abe1-413e0bed856a.png)

Reply:
- [ ] @kidde60 great work implementing the changes requested however I can't manage to store the post created in the database, kindly check your post model you can try deleting it and adding it again using `rails g model model-name`
![image](https://user-images.githubusercontent.com/61408860/206849255-776c985e-2e24-4bf5-b12b-e65574cd6300.png)

Reply:
- [ ] @kidde60 great work trying to fix the problem, but the problem is still there, I would suggest you relook at how you create your migrations the schema file shows you are referencing the users_id, while in your model it's referencing the author_id if it still doesn't work kindly contact me on slack or ask for assistance.